.pre-commit-config.yaml

```diff
@@ -40,7 +40,7 @@ repos:
       - id: migrations-check
         language: system
         name: Check for uncreated migrations.
-        entry: bash -c "./manage.py makemigrations --check --dry-run"
+        entry: sh -c "./manage.py makemigrations --check --dry-run"
```

Comment:
hey~ just a question. what is the advantage of using `sh` over `bash`?

Reply:
Hi,
sh is used more widely, in old days /bin/sh was linked to /bin/bash but nowadays some distros/OSes do not ship with /bin/bash and /bin/sh can link e.g. to /bin/dash.
That is the case for me, FreeBSD does not ship with bash by default.